src/main/kotlin/it/pagopa/ecommerce/helpdesk/utils/ResponseBuilderV2.kt

```diff
@@ -69,6 +71,16 @@ fun baseTransactionToTransactionInfoDtoV2(
                     UUID.fromString(transactionGatewayActivationData.correlationId)
                 else null
             )
+            .gatewayAuthorizationStatus(
```

Comment:
those fields can be populated also for PGS and Redirect gateway, I would extract this to a separated method and return gateway status and error codes for all transactions rather than returning those fields only for NPG ones

Reply:
The informations can be taken from the respective transactionGatewayAuthorizationData instances

Reply:
Fixed with 0e2f9f2b0afc2be717539ce7772d65231e789cca